Asterisk.2013/Asterisk.NET/Manager/Action/ParkAction.cs

```diff
@@ -50,7 +54,12 @@ public override string Action
         /// <summary>
         ///     Set the Channel where the Call will end up after the timeout is reached.

```

Comment:
Maybe add the full parameter description shown on the Asterisk Documentation:
https://wiki.asterisk.org/wiki/display/AST/Asterisk+12+ManagerAction_Park
> If specified, then this channel will receive an announcement when Channel is parked if AnnounceChannel is in a state where it can receive announcements (AnnounceChannel must be bridged). AnnounceChannel has no bearing on the actual state of the parked call.